tools/blast_rbh/.shed.yml

```diff
@@ -13,20 +13,20 @@ type: unrestricted
 include:
 - strip_components: 2
```

Comment:
Should `strip_components` be removed?

Reply:
Yes - it should

Reply:
Cross reference https://planemo.readthedocs.io/en/latest/standards/docs/best_practices/shed_yml.html#shed-upload-includes-excludes which would lose its example use case ;)